src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -49,6 +59,32 @@ def tearDown(self):
 
     def run_script(self, argument, check_stderr=False, verbose=False):
         print('\n    Running sonic-cfggen ' + argument)
+        if PY3x and "-m" in argument:
```

Comment:
>argument [](http://example.com/codeflow?start=28&length=8)

Suggest use `ArgumentParser` and `shlex.split()` to parse the arguments in a string. ref: [https://stackoverflow.com/a/18596435/2514803](https://stackoverflow.com/a/18596435/2514803)